.github/workflows/labeler.yml

```
@@ -0,0 +1,15 @@
+name: "Pull Request Labeler"
+on:
+    pull_request_target:
+
+jobs:
+  labeler:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)